### PR TITLE
Add custom and user defined event

### PR DESCRIPTION
**Pull request by omarmosid**

### CODE REVIEW

**Comment by omarmosid** [comment]
closes #1

**Comment by ad-astra-via** [comment]
@omarmosid I think we are missing beside custom and user-defined events the following:

- Search
- ViewCategory

List here: https://help.pinterest.com/en/business/article/add-event-codes

Let me know if you want me to add it + test it!

File: src/index.ts [hunk 1]
```diff
@@ -106,7 +106,23 @@ export default async function (manager: Manager, settings: ComponentSettings) {
     handler('signup', event, settings)
   })
 
-  manager.addEventListener('watchVideo', event => {
+  manager.addEventListener('viewcategory', event => {
+    handler('viewcategory', event, settings)
+  })
+  manager.addEventListener('watchvideo', event => {
```
Comment (tomkln) [inline src/index.ts:112]: Should those be all lowercase?

Comment (ad-astra-via) [reply src/index.ts:112]: @tomkln are you referring to the event names "viewcategory", "watchvideo", etc? If so then yes.

Comment (tomkln) [reply src/index.ts:112]: Awesome, looks good to me then :)

**Review by tomkln — APPROVED** [review]

**Pull request merged**